alma/alma-woocommerce-gateway
# add loader between modal

## Reason for change



[Add loader](https://linear.app/almapay/issue/MPP-522/add-loader-between-modal)

### Code changes



Add loader wrapper in js code

### How to test

_As a reviewer, you are encouraged to test the PR locally._

Open modal to In-Page and see if loader is well displayed et closed if we closed the modal

### Checklist for authors and reviewers




### Non applicable
